lua/scratch/config.lua

```diff
@@ -128,6 +128,27 @@ M.checkInit = function()
   return vim.fn.filereadable(CONFIG_FILE_FLAG_PATH) == 1 and validate_abspath(getConfigFilePath())
 end
 
+---@param configFilePath string
+local function writeConfig(configFilePath)
+  -- write the scratch_file_dir into CONFIG_FILE_PATH file
+  local dir_path = vim.fn.fnamemodify(CONFIG_FILE_FLAG_PATH, ":h")
+  if vim.fn.isdirectory(dir_path) == 0 then
+    vim.fn.mkdir(dir_path, "p")
+  end
+
+  local file = io.open(CONFIG_FILE_FLAG_PATH, "w")
+  file:write(configFilePath)
+  file:close()
```

Comment:
This needs to be in the main function (`initProcess`), otherwise it will keep asking the user every time vim starts.